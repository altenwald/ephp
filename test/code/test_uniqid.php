<?php

print "uniqid() = " . uniqid() . "\n";
print "uniqid('x') = " . uniqid('x') . "\n";
print "uniqid('', true) = " . uniqid('', true) . "\n";
print "uniqid('y', true) = " . uniqid('y', true) . "\n";

