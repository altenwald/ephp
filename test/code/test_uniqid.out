uniqid() = 53d5415caa0d8
uniqid('x') = x53d5415caa0d8
uniqid('', true) = 53d5415caa0d80.72643441
uniqid('y', true) = y53d5415caa0d80.72643441
